packages/sampler/stories/qa/Scroller.js

```diff
@@ -464,6 +533,9 @@ storiesOf('Scroller', module)
 						direction="horizontal"
 						horizontalScrollbar="visible"
 						noScrollByWheel={noScrollByWheel}
+						onKeyDown={action('onKeyDown (nested 2nd scroller)')}
```

Comment:
I think the first letters of `nested` and `scroller` would be Upper case because `n` is the first letter in the sentence and `Scroller` is the component name. How about your thoughts?

Reply:
I also tried just like you said but I felt capitalization looks like to make the word emphasized. 
whereas the event type is starting lowercase. the contents inside the parenthesis are the purpose of giving information to a tester. If the expressions look wired or make confusing to others I'll fix them.